app/services/attachment/new_service.rb

```diff
@@ -14,7 +14,7 @@ def call
   attr_reader :user, :manual_id, :section_uuid
 
   def attachment
-    Attachment.new(initial_params)
+    Attachment.new({})
```

Comment:
Minor: I think this could be further simplified to `Attachment.new`, because `Attachment` is a `Mongoid::Document` and so an empty Hash of attributes is the default.